std/conv.d

```diff
@@ -3671,38 +3671,26 @@ of the passed-in integral.
 See_Also:
     $(LREF parse) for parsing octal strings at runtime.
  */
-@property int octal(string num)()
-    if((octalFitsInInt!(num) && !literalIsLong!(num)) && !literalIsUnsigned!(num))
-{
-    return octal!(int, num);
-}
-
-/// Ditto
-@property long octal(string num)()
-    if((!octalFitsInInt!(num) || literalIsLong!(num)) && !literalIsUnsigned!(num))
-{
-    return octal!(long, num);
-}
-
-/// Ditto
-@property uint octal(string num)()
-    if((octalFitsInInt!(num) && !literalIsLong!(num)) && literalIsUnsigned!(num))
-{
-    return octal!(int, num);
-}
-
-/// Ditto
-@property ulong octal(string num)()
-    if((!octalFitsInInt!(num) || literalIsLong!(num)) && literalIsUnsigned!(num))
-{
-    return octal!(long, num);
+template octal(string num)
```

Comment:
Note that this change from 4 to 1 symbols was made because when `octal!foo` is no longer a function, its return type is not shown in documentation, so bundling them together produces better documentation. The return type logic is described in the documentation comment.